### ISSUE TITLE
APIM-CLI Fails to export API WSDL Specifications starting with comments

### ISSUE THREAD

**Issue by fbessai** [Axway-API-Management-Plus/apim-cli#364]
### APIM-CLI version

1.13.3

### API-Management version

7.7.0.20220228

### Bug description

When the WSDL content starts with comments, apim-cli fails to recognise it as a WSDL specification.

It appears from the code source that only the  first 500 characters of the WSDL content are actually parsed. In case the WSDL starts with  comments, the first 500 might not contain the key word "wsdl" which causes class "com.axway.apim.api.apiSpecification.WSDLSpecification" to reject it as a WSDL content.

Suggestion  :  increase the 500 chars limit or make that limit a variable that can be set in the configuration/params

### Steps to reproduce

1. Import a WSDL Specification API with API Manager UI ( use any wsdl content that starts with a big comment ex : "<?xml version='1.0' encoding='UTF-8'?><!-- Generated by JAX-WS RI at http://jax-ws.dev.java.net. RI's version is JAX-WS RI 2.2.6b21  svn-revision#12959. --")
2.  Export this API with apim-cli 

### Relevant log output

```shell
023-03-13 20:05:15,042 [APISpecificationFactory] DEBUG: Handle API-Specification: <?xml version='1.0' encoding='UTF-8'?><!-- Published by JAX-WS RI at http://jax-ws.dev.java.net. RI's version is JAX-WS RI 2.2.6b21  svn-revision#12959. --><!-- Generated by JAX-WS RI at http://jax-ws... , apiDefinitionFile: api_XXXXXXXXXX_2_2023-03-13T19:05:15Z.xml , API Name : API_XXXXXXXXXXX_2
2023-03-13 20:05:15,056 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.apiSpecification.Swagger2xSpecification 
2023-03-13 20:05:15,074 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.apiSpecification.Swagger1xSpecification 
2023-03-13 20:05:15,075 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.apiSpecification.OAS3xSpecification
2023-03-13 20:05:15,077 [WSDLSpecification] DEBUG: No WSDL specification. Specification doesn't contain wsdl in the first 500 characters.
2023-03-13 20:05:15,078 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.apiSpecification.WSDLSpecification
2023-03-13 20:05:15,079 [WADLSpecification] DEBUG: No WADL specification. Specification doesn't contain WADL namespace: wadl.dev.java.net in the first 500 characters.
2023-03-13 20:05:15,080 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.apiSpecification.WADLSpecification
```

**Comment by rathnapandi** [comment]
Hi @fbessai, 
Fixed the code by parsing the xml and looking for wsdl namespace. 
Please test your use case with binary - 
[axway-apimcli-1.13.5-SNAPSHOT.tar.gz](https://github.com/Axway-API-Management-Plus/apim-cli/files/10961784/axway-apimcli-1.13.5-SNAPSHOT.tar.gz)

**Comment by fbessai** [comment]
Hi @rathnapandi , 
I'm still the exact same issue with this 1.13.5 snapshot version.

`
2023-03-14 15:25:33,979 [APIManagerCLI] INFO : API-Manager CLI: 1.13.5-SNAPSHOT
2023-03-14 15:25:33,979 [APIManagerCLI] INFO : Module: API - E X P O R T / U T I L S (1.13.5-SNAPSHOT)
2023-03-14 15:25:33,979 [EnvironmentProperties] DEBUG: Trying to load environment properties from file: env.properties ... not found.
2023-03-14 15:25:34,950 [APIMHttpClient] DEBUG: API Manager CLI timeout : 30000

...
...
...
2023-03-14 15:25:36,918 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.specification.Swagger2xSpecification
2023-03-14 15:25:36,918 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.specification.Swagger1xSpecification
2023-03-14 15:25:36,918 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.specification.OAS3xSpecification
2023-03-14 15:25:36,918 [WSDLSpecification] DEBUG: No WSDL specification. Specification doesn't contain wsdl in the first 500 characters.
2023-03-14 15:25:36,918 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.specification.WSDLSpecification
2023-03-14 15:25:36,918 [WADLSpecification] DEBUG: No WADL specification. Specification doesn't contain WADL namespace: wadl.dev.java.net in the first 500 characters.
2023-03-14 15:25:36,918 [APISpecificationFactory] DEBUG: Can't handle API specification with class: com.axway.apim.api.specification.WADLSpecification`

**Comment by rathnapandi** [comment]
@fbessai 
Sorry, I did upload the wrong file. I have tested it with attached binary. 
[apimcli-13.5.-test.tar.gz](https://github.com/Axway-API-Management-Plus/apim-cli/files/10970244/apimcli-13.5.-test.tar.gz)
Can you validate it>

**Comment by fbessai** [comment]
@rathnapandi , 
It issue is resolved with this version. 
Thank you very much for the promptness of your help.
Do you have an idea when this gets released ?

**Comment by rathnapandi** [comment]
closed as part of 1.13.5

**Comment by fbessai** [comment]
@rathnapandi , thanks you very much
